### PR TITLE
feat(web): improvements to slideshow

### PR DESCRIPTION
**Pull request by martabal**
## What's changed

This PR adds multiple improvements to the slideshow mode:
- Hide cursor and slideshow bar after 10s of inactivity
- ~Hide the video controls if the slideshow is in full screen~
- Add an option to go back to full screen
- ~Play and Pause video with the slideshow play / pause button~

### CODE REVIEW

File: web/src/lib/components/asset-viewer/asset-viewer.svelte [hunk 1]
```diff
@@ -188,7 +190,12 @@
     }
   }
 
+  const handleFullScreenChange = () => {
+    isFullScreen = document.fullscreenElement !== null;
```
Comment (jrasm91) [inline web/src/lib/components/asset-viewer/asset-viewer.svelte:194]: https://svelte.dev/docs/special-elements#svelte-document

Comment (martabal) [reply web/src/lib/components/asset-viewer/asset-viewer.svelte]: Damn, I always forget about that one, thanks !

**Comment by jvgoch** [comment]
Hey man, don't know if this is because of my feature request, but thnx for taking the time to do this!

**Comment by alextran1502** [comment]
Hello, I just tested the PR, and everything works well besides the `Play and Pause video with the slideshow play/pause button.` When playing a video in slide show mode, clicking to pause the slideshow doesn't stop the progress bar from running and advancing to the next asset. I would like to revert this mechanism and have the slideshow control not interfere with the video control.

**Review by alextran1502 — APPROVED** [review]

**Pull request merged**